doc/user/install-operator-sdk.md

```diff
@@ -75,12 +75,23 @@ $ brew install operator-sdk
 
 ## Compile and install from master
 
+### Prerequisites
+
+- [git][git_tool]
+- [mercurial][mercurial_tool] version 3.9+
+- [bazaar][bazaar_tool] version 2.7.0+
```

Comment:
Does the go tool-chain need all 3 vcs tools to be installed? First time I'm seeing bazaar.
https://github.com/golang/go/wiki/GoGetTools

Reply:
Yeah, we need git, mercurial, and bazaar, since somewhere in our dependency tree there are deps that use them. Nothing uses SVN at the moment, so we don't need that one.